microservices/services/query-metric/service/src/test/java/datawave/microservice/querymetric/QueryMetricSplitBrainMergePolicyTest.java

```diff
@@ -33,7 +33,7 @@ public class QueryMetricSplitBrainMergePolicyTest {
     
     private final static QueryMetricFactory queryMetricFactory = new QueryMetricFactoryImpl();
     
-    @Ignore
+    @Disabled
```

Comment:
Why does this show up?  This was already done in integration.

Reply:
I made this change before integration was fixed. This will get sync'ed when I rebase the branch, will do that today.